packages/perseus/src/util.ts

```diff
@@ -9,6 +9,7 @@ import type {Range} from "./perseus-types";
 import type {PerseusStrings} from "./strings";
 import type {Widget, PerseusScore} from "./types";
 import type {KEScore} from "@khanacademy/perseus-core";
+import type * as React from "react";
```

Comment:
We could also add `@types/react` as a dev dependency 
https://www.npmjs.com/package/@types/react

Reply:
We do have that as a dependency in the root `package.json` (https://github.com/Khan/perseus/blob/main/package.json#L61) :)